Source/Scene/Cesium3DTileFeature.js

```diff
@@ -92,17 +116,301 @@ define([
          */
         color : {
             get : function() {
+                if (defined(this._labelCollection)) {
+                    var label = this._labelCollection.get(this._batchId);
```

Comment:
What is this `get` function needed?  Why can't we always keep reference to the label?